inc/utilities/class-oauth2-wp-cli.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace WP\OAuth2\Utilities;
+
+class Oauth2_Wp_Cli extends \WP_CLI_Command {
```

Comment:
This class should just be called something like `Command` instead, no need to prefix inside the namespace. Also, `WP_CLI_Command` should be `use`'d at the top of the file rather than using an absolute reference.